aries-site/src/pages/foundation/voice-and-tone.mdx

```diff
@@ -1,54 +1,28 @@
 import { Anchor, Box, Card, Text } from 'grommet';
 import { StatusCritical, StatusInfo } from 'grommet-icons';
 
-Messaging within HPE applications should be informative, concise, and trustworthy.
-It should be delivered in active, positive, objective, conversational
-American English and written from a third-person [point of view](#point-of-view).
-Executed properly, messaging within HPE applications should leave users
-feeling and perceiving HPE and its products to be intelligent, experienced,
-inclusive, and trustworthy.
+The HPE Design System employs text-based content that cultivates an exceptional user experience.
```

Comment:
I'm wondering if "text-based content" is too formal. Something like "words" feels more human to me. But, if it's a standard term that I might not be aware of, OK.

Reply:
Good call. I'll update this and try to underscore that content equals words.